gwebsockets/server.py

```diff
@@ -18,7 +18,7 @@
 if sys.version_info[0] < 3:
     from StringIO import StringIO
 else:
-    from io import StringIO
+    from io import BytesIO
 from collections import deque
```

Comment:
This should possibly create a conflict. StringIO is not used, BytesIO is not imported in Python2, we can remove python2 version right?